### PR TITLE
Add ACX Collateral UMIP

### PR DESCRIPTION
**Pull request by mrice32**

### CODE REVIEW

File: UMIPs/umip-167.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+## Motivation
+
+The addition of this collateral currency offers additional functionality to the UMA protocol and increases the range of contracts that can be built.
+
+## Technical Specification
+
+To accomplish this upgrade, the following changes need to be made:
+
+-   The ACX address [0x44108f0223A3C3028F5Fe7AEC7f9bb2E66beF82F](https://etherscan.io/address/0x44108f0223A3C3028F5Fe7AEC7f9bb2E66beF82F) needs to be added to the collateral currency whitelist introduced in UMIP-8.
```
Comment (Reinis-FRP) [inline UMIPs/umip-167.md:22]: Is it sufficient to add mainnet ACX or there are known L2 deployments that could be added in this UMIP as well

Comment (mrice32) [reply UMIPs/umip-167.md:22]: For the LSP deployments on mainnet, this should be sufficient. Do we typically add collateral currencies everywhere regardless of where the planned use is?

File: UMIPs/umip-167.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+## Headers
+
+| UMIP-167            |                                                      |
+| ------------------- | ---------------------------------------------------- |
+| UMIP Title          | Add ACX as a supported collateral currency           |
+| Authors             | Matt Rice (matt@umaproject.org)                   |
+| Status              | Draft                                                |
+| Created             | 2022-11-22                                           |
+
```
[the remark this answers is not in the document]
Comment (mrice32) [reply UMIPs/umip-167.md:10]: Done!

**Review by smb2796 — APPROVED** [review]

File: UMIPs/umip-167.md (hunk 1 above)
Comment (mrice32) [reply UMIPs/umip-167.md:23]: I agree with your point @Reinis-FRP that a final fee update may be necessary in the future, and I would also like to keep this simple given the timeline, so maybe it's best to just have it be approved on mainnet for now?

Comment (Reinis-FRP) [reply UMIPs/umip-167.md:23]: agree

**Review by Reinis-FRP — APPROVED** [review]

**Pull request merged**